ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -320,42 +319,40 @@ class VariantSearchFormPanels extends React.PureComponent {
           </ButtonLink>
         </ExpandCollapseCategoryContainer>
         <VerticalSpacer height={10} />
-        <FormSection name="search">
```

Comment:
this didn't change, it just unindented since its no linger nested in the FormSection